xml/System.IO/FileSystemInfo.xml

```diff
@@ -1301,10 +1301,12 @@ On Unix platforms that do not support creation or birth time, this property retu
         <remarks>
           <format type="text/markdown"><![CDATA[  
   
-## Remarks  
- `FileSystemInfo.Refresh` takes a snapshot of the file from the current file system. `Refresh` cannot correct the underlying file system even if the file system returns incorrect or outdated information. This can happen on platforms such as Windows 98.  
-  Calls must be made to `Refresh` before attempting to get the attribute information, or the information will be outdated.  
-  
+## Remarks
+
+`FileSystemInfo.Refresh` takes a snapshot of the file from the current file system.
```

Comment:
The way the old sentence was worded made me wonder if the Win 98 remark was referring to the two sentences that preceded it or only the latter. So I looked at the code to confirm that your edit is correct: `Refresh` internally calls the [FillAttributeInfo method](https://github.com/dotnet/runtime/blob/79ae74f5ca5c8a6fe3a48935e85bd7374959c570/src/libraries/Common/src/System/IO/FileSystem.Attributes.Windows.cs#L54) and simply takes a snapshot.